src/main/java/org/osc/controller/nsc/entities/PortEntity.java

```diff
@@ -83,6 +83,14 @@ public NetworkElementEntity(String elementId, List<String> macAddressEntities, L
         this.portIPs = portIpEntities;
     }
 
+    public PortEntity(PortEntity element){
+        this.elementId = element.elementId;
+        this.deviceOwnerId = element.deviceOwnerId;
+        this.parentId = element.parentId;
```

Comment:
if parentid is not applicable, can we just remove it?

Reply:
parentId access methods are needed, as it is defined in the Base Class.